internal/cli/admin_opts.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
[nit] can we move this to the `/opsmanager/admin/backup` package? this way the name is less confusing, `Admin` is to broad of a term to be in the global `cli` package

Reply:
In this way we are going to hit `import cycle not allowed`. Should I rename it as `create_opts`?

Reply:
feel free then to create a `backupstore` package in the admin one